game-core/src/main/java/games/strategy/triplea/ui/menubar/help/UnitStatsTable.java

```diff
@@ -103,15 +103,13 @@ public static String getUnitStatsTable(final GameData gameData, final UiContext
     }
     final Set<UnitType> all = data.getUnitTypeList().getAllUnitTypes();
     all.removeAll(unitsSoFar);
+    unitTypes.put(
+        GamePlayer.NULL_PLAYERID,
+        getPlayerUnitsWithImages(GamePlayer.NULL_PLAYERID, data, uiContext));
+    unitsSoFar.addAll(unitTypes.get(GamePlayer.NULL_PLAYERID));
+    all.removeAll(unitsSoFar);
     if (!all.isEmpty()) {
```

Comment:
Could you explain why this if statement is always true?
If all is no subset of unitsSoFar this isn't always the case